socorro/processor/mozilla_processor_2015.py

```diff
@@ -63,6 +63,7 @@
         "socorro.processor.signature_utilities.SignatureRunWatchDog, "
         "socorro.processor.signature_utilities.SignatureIPCChannelError, "
         "socorro.processor.signature_utilities.SignatureIPCMessageName, "
+        "socorro.processor.signature_utilities.SigTrim, "
```

Comment:
I really don't understand the logic of putting this transform *before* `SigTrunc`. Suppose it's 

1) `Some signature that is really long `
2) `Some signature that is really long`
2) `Some signature that ` (i.e. truncating away the last couple of words)